wisdem/glue_code/gc_WT_DataStruc.py

```diff
@@ -2161,7 +2161,7 @@ def setup(self):
             ivc.add_output("ring_stiffener_web_thickness", 0.0, units="m")
             ivc.add_output("ring_stiffener_flange_width", 0.0, units="m")
             ivc.add_output("ring_stiffener_flange_thickness", 0.0, units="m")
-            ivc.add_output("ring_stiffener_spacing", 0.0, units="m")
+            ivc.add_output("ring_stiffener_spacing", 0.0)
```

Comment:
Do the units have to come off the axial_stiffener_spacing a few lines below too?

Reply:
Maybe nevermind.  It seems like we are keeping those in deg/rad

Reply:
Right, we could change it to a 0 to 1 non-dimensionalization in line with the ring stiffeners, though radians doesn't vary with diameter whereas the ring_stiffener spacing would vary with member length. Do you prefer that we change this, or is it good to keep it in deg/rad?

Reply:
It's okay to keep in deg/rad.